Readme.md

```diff
@@ -85,9 +87,7 @@ export default withUrlState<OwnProps, LiftedState>(history, (prop: OwnProps) =>
 
 ## Motivation
 
-Being able to have a sharable link which captures the state of a page can be very useful functionality for users.
-
-This commonly occurs when viewing search results, filtering or querying over a data set or even tracking the currently visible portion on a map.
+A link with a specific state embedded into the url is useful for querying data or viewing search results. 
```

Comment:
How about: 

When you need to have a url which updates based on search results, filtering or querying a data set or even tracking the visible portion of a map `with-url-state` can do the tedious query parameter manipulation for you, making it easy to have sharable links for your users

The map bit was actually one of my main use cases!

Reply:
Sure, although personally I'm against using the second person point of view. Here's my take on the motivation section:

`with-url-state` automates the query parameter manipulations, simplifying URL sharing for search results, querying data or tracking a visible portion of a map.

Reply:
👍